src/GraphQL/Mutations.tsx

```diff
@@ -0,0 +1,31 @@
+import { gql } from "@apollo/client";
```

Comment:
I would name the folder `graphql` or maybe `graph-ql` to be consistent with the other folders